validation/views.py

```diff
@@ -928,3 +942,26 @@ def create_new_rec_id(phrase, speaker):
         f"{time.time()}\n"
     )
     return sha256(signature.encode("UTF-8")).hexdigest()
+
+
+def save_metadata_to_file(rec_id, user, transcription, translation):
+    dest = (
+        settings.MEDIA_ROOT
+        + "/"
+        + settings.RECVAL_AUDIO_PREFIX
+        + "metadata/"
```

Comment:
Would this directory need to be created, or does it already exist?

Reply:
I just made it on the server, so everything should be good to go now